utils/handler.ts

```diff
@@ -73,3 +73,25 @@ export const handleCopy = async (value: string) => {
 export const handleNftImageLoadError = (e: React.SyntheticEvent<HTMLImageElement, Event>) => {
   e.currentTarget.src = '/images/nft-placeholder.svg'
 }
+
+export const handleSorterArrayFromUrl = (url, sorters) => {
+  const paramArr = url.slice(url.indexOf('?') + 1).split('&')
```

Comment:
Use `URLSearchParams` to parse query

Ref: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams